scaleway/resource_k8s_pool_beta.go

```diff
@@ -185,6 +185,10 @@ func resourceScalewayK8SPoolBetaRead(d *schema.ResourceData, m interface{}) erro
 	d.Set("created_at", pool.CreatedAt)
 	d.Set("updated_at", pool.UpdatedAt)
 
+	if pool.PlacementGroupID != nil {
+		d.Set("placement_group_id", newRegionalId(region, *pool.PlacementGroupID))
```

Comment:
placement_group_id should not be regional

Reply:
same :p